source/_integrations/russound_rio.markdown

```diff
@@ -18,4 +18,9 @@ The `russound_rio` {% term integration %} allows you to control Russound devices
 
 The platform automatically discovers all enabled zones and sources. Each zone is added as a media player device with the enabled sources available as inputs. Media information is supported if the selected source reports it.
 
+{% important %}
+If you want to control the Russound ACA, CAV, CAM, CAA, and CAS controllers,
+check out the [Russound RNET](/integrations/russound_rnet/) integration.
+{% endimportant %}
+
 {% include integrations/config_flow.md %}
```

Comment:
```suggestion
{% include integrations/config_flow.md %}

## Supported devices

This integration allows you to control the Russound MCA, MBX, XSource, XZone, and XStream controllers.

If you are looking to control Russound ACA, CAV, CAM, CAA, and CAS controllers, refer to the [Russound RNET](/integrations/russound_rnet/) integration

```
I think this could go into the supported devices section, like we have with other integrations.

Reply:
Yep, sounds good.